portal/server/constants/field-names.js

```diff
@@ -11,14 +11,14 @@ const DEAL = {
 
 const FACILITY = {
   TYPE: 'type',
-  HAS_BEEN_ISSUED: 'hasBeenIssued',
   NAME: 'name',
   UKEF_FACILITY_ID: 'ukefFacilityId',
   CURRENCY: 'currency',
   VALUE: 'value',
   DEAL_SUBMISSION_TYPE: 'deal.submissionType',
   EXPORTER_COMPANY_NAME: 'deal.exporter.companyName',
   CREATED_BY: 'createdBy',
+  STAGE: 'stage',
```

Comment:
This entire area of the code is really rather confusing. Would it be possible to add a comment to the top of this file as to the use of this file (at a glance, it seems to be used in collab with content-strings?)

Not for this PR - This file adds an additional layer of (in my opinion) probably unneeded abstraction. We're tying our types to unchecked strings -- ie if we changed companyName, we'd probably miss updating it here... This is really rather confusing, great job navigating this.